workflow/emc-global/scripts/exglobal_prep_chem.sh

```diff
@@ -147,10 +147,10 @@ cd ..
 # Any variables have to be exported to the environment before substitution
 if [[ $SHOUR == 00 ]] ; then
     export GBBEPX_DATA_DIR=$GBBEPX_DATA_DIR_YESTERDAY
-    export gbbepx_days=${gbbepx_days:-'$PDYm2 $PDYm3'}
+    export gbbepx_days=${gbbepx_days:-'$PDYm2'}
 else
     export GBBEPX_DATA_DIR=$GBBEPX_DATA_DIR_TODAY    
-    export gbbepx_days=${gbbepx_days:-'$PDYm1 $PDYm2'}
```

Comment:
In the previous version, you use to PDYs for gbbepx_days, so it will get two files, such as below, however, when you delete one PDY, it should only find one type of files. I don't know whether it will cause some difference or not.  

/lfs/h1/ops/prod/dcom/20220818/firewx
-rw-rw-r-- 1 dfprod prod 589832 Aug 17 20:00 GBBEPx.emis_BC.003.20220816.FV3.C384Grid.tile1.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 17 20:00 GBBEPx.emis_BC.003.20220816.FV3.C384Grid.tile2.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 17 20:00 GBBEPx.emis_BC.003.20220816.FV3.C384Grid.tile3.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 17 20:00 GBBEPx.emis_BC.003.20220816.FV3.C384Grid.tile4.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 17 20:00 GBBEPx.emis_BC.003.20220816.FV3.C384Grid.tile5.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 17 20:00 GBBEPx.emis_BC.003.20220816.FV3.C384Grid.tile6.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 18 01:30 GBBEPx.emis_BC.003.20220817.FV3.C384Grid.tile1.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 18 01:30 GBBEPx.emis_BC.003.20220817.FV3.C384Grid.tile2.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 18 01:30 GBBEPx.emis_BC.003.20220817.FV3.C384Grid.tile3.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 18 01:30 GBBEPx.emis_BC.003.20220817.FV3.C384Grid.tile4.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 18 01:30 GBBEPx.emis_BC.003.20220817.FV3.C384Grid.tile5.bin
-rw-rw-r-- 1 dfprod prod 589832 Aug 18 01:30 GBBEPx.emis_BC.003.20220817.FV3.C384Grid.tile6.bin

How about the below changes:

in j-jobs:
  export GBBEPX_DATA_DIR_TODAY=${GBBEPX_DATA_DIR_TODAY:-$DCOMROOT}
  export GBBEPX_DATA_DIR_YESTERDAY=${GBBEPX_DATA_DIR_YESTERDAY:-$DCOMROOT}

in the ex-script:

  gbbepx_pattern=${gbbepx_pattern:-'$GBBEPX_DATA_DIR/${day}/firewx/${local_name}.003.${day}.FV3.${CASE}Grid.${tiledir}.bin'}

Reply:
@XianwuXue-NOAA 
after GB Input changes: 
For example today is 20221012 (PDY=20221012), but GBBEPX_DATA_DIR_TODAY actually points to 20221011 (PDYm1), which is firewx/20221011. Since firewx/20221012 is unavailable;  Therefore, GBBEPX_DATA_DIR_YESTERDAY points to 20221010 (PDYm2).